modules/menus/audio/active/SelectedPlayback.ts

```diff
@@ -35,25 +38,27 @@ const renderActivePlayback = (): Box<Child, Attribute>[] => {
                 }),
                 Widget.Box({
                     vertical: true,
-                    children: [
-                        Widget.Label({
-                            class_name: 'menu-active playback',
-                            hpack: 'start',
-                            truncate: 'end',
-                            expand: true,
-                            wrap: true,
-                            label: audio.bind('speaker').as((v) => v.description || ''),
-                        }),
-                        Widget.Slider({
-                            value: audio['speaker'].bind('volume'),
-                            class_name: 'menu-active-slider menu-slider playback',
-                            draw_value: false,
-                            hexpand: true,
-                            min: 0,
-                            max: 1,
-                            onChange: ({ value }) => (audio.speaker.volume = value),
-                        }),
-                    ],
+                    children: raiseMaximumVolume.bind('value').as((raiseMaxVol) => {
```

Comment:
Could we move this binding down to the property reliant upon it? In this case the max property, to some thing like:
```ts
                            max: raiseMaximumVolume.bind('value').as((raiseMax) => (raiseMax ? 1.5 : 1)),

```

Reply:
Generally I've notice stacking behavior when you put a binding inside a binding (our top level raiseMaxVol binding nesting the `audio.bind('speaker')`.

This way only the properties truly relying on the value are bounded.